plugins/datadog/app/models/datadog_notification.rb

```diff
@@ -20,8 +20,8 @@ def deliver
       event_object: Digest::MD5.hexdigest("#{Time.new}|#{rand}"),
       alert_type: status,
       source_type_name: "samson",
-      date_happened: @deploy.updated_at,
-      tags: @stage.datadog_tags_as_array + ["deploy"]
+      date_happened: now ? Time.now : @deploy.updated_at,
```

Comment:
Alternative would be to create a new hook `:during_deploy` in addition to the existing `:before_deploy` and `:after_deploy` to use the `@deploy.updated_at` but this seemed sufficient / more clean and reusable for an event timestamp.